service_name: "~NeckSetAngle"

std_vertical_angle: 0
std_horizontal_angle: 0

servo_1_channel: 9
servo_2_channel: 8

servo_1_coef: 0
servo_2_coef: 0

constraints:
  v_from: -30
  v_to: 30
  h_from: -30
  h_to: 30